app/models/registerable_edition.rb

```diff
@@ -14,7 +14,7 @@ def slug
     # to be consistent with Panopticon's slug format.
     panopticon_slug = Whitehall.url_maker.public_document_path(edition).sub(/\A\//, "")
     if edition.deleted?
-      panopticon_slug.sub!(%r{/deleted-([^/]+)$}, '/\1')
+      panopticon_slug.sub!(%r{deleted-([^/]+)$}, '\1')
```

Comment:
ah, so the regexp matching `/` at the start was a deliberate attempt to anchor it, but just didn't take into account detailed guides which can appear at the root of the website.

Maybe something like this

```ruby
panopticon_slug.sub!(%r{(^|/)deleted-([^/]+)$}, '\1\2')
```